certify-service/src/main/resources/5b9c2a12-810a-7388-2dc8-13ee7ad88bac

```diff
@@ -0,0 +1,61 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="250" height="400" viewBox="0 0 250 400">
```

Comment:
Once we receive the files for Insurance & Mock, we should add these to the docker-compose as well.

Reply:
Will add similiar configurations in Insurance and Mock once we receive the files